src/resolvers/collection.js

```diff
@@ -135,18 +139,20 @@ export default {
       }
 
       return dataSources.toolSource({
-        conceptId: tools,
+        conceptId: toolConceptIds,
         ...handlePagingParams(args, tools.length)
       }, context, parseResolveInfo(info))
     },
     variables: async (source, args, context, info) => {
       const {
-        associations = {}
+        association_details: associationDetails = {}
       } = source
 
       const { dataSources } = context
 
-      const { variables = [] } = associations
+      const { variables = [] } = associationDetails
+
+      const variableConceptIds = variables.map((variable) => variable.concept_id)
```

Comment:
This could be simplified a bit with some destructuring

```suggestion
      const variableConceptIds = variables.map(({ concept_id }) => concept_id)
```

Reply:
Using de-structuring to retrieve the field, thank you!